File: programme/radian-to-degree/radian-to-degree.js
// Write a programme to convert radian to degree

"use strict";

function radian_to_degree(radian) {
  var pi = Math.PI;
  return radian * (180 / pi);
}

let radianInput = prompt("Radial : ");

console.log("Input  : " + radianInput);
console.log("Output : " + radian_to_degree(radianInput).toFixed(5));

File: programme/radian-to-degree/radian-to-degree.go
// Write a programme to convert radian to degree
package main

import (
	"fmt"
	"math"
)

func main() {
	var radian float64
	fmt.Print("Enter a radian : ")
	fmt.Scan(&radian)
	fmt.Println("\nDegree of", radian, "radian is", math.Round(radian*180/math.Pi))
}
